galaxy_ng/app/api/v3/urls.py

```diff
@@ -53,4 +53,10 @@
         viewsets.CollectionArtifactDownloadView.as_view(),
         name='collection-artifact-download'
     ),
+    path(
+        'collections/<str:namespace>/<str:name>/versions/<str:version>/move/'
+        '<str:source_path>/<str:dest_path>/',
```

Comment:
Wondering if there is any utility to making it explicit that this is an 'action'. 

https://geemus.gitbooks.io/http-api-design/content/en/requests/actions.html suggests that. I can see the reasoning behind the suggestion and generally like the rest of the guidelines/suggestions in that doc. 

But it also seems like it doesn't matter that much.

Reply:
I'm less sure about the /move/{src}/{dest}  path args. On one hand, it's straightforward and elegant. On the other hand, /move/{src}/  kind of prevents there ever being /move/ sub actions  (/move/recursive/src/dest maybe?).  Probably not a big deal, since there could always be another 'action'  (ie, /move_recursive/{src}/{dest}/).

I guess one concern would be if the 'src' and 'dest' would ever need to be 'pulp_href's which could be complicated
(ie, /move/pulp/api/v3/collections/ansible/ansible/XXXXXX/pulp/api/v3/collections/ansible/ansible/YYYY).

Reply:
Looking at pulp_rpm's 'copy' endpoint (https://pulp-rpm.readthedocs.io/en/latest/restapi.html#tag/rpm:-copy)  and it is just `/pulp/api/v3/rpm/copy/`.

So, /move/ seems okay.

But then again, some pulp plugin API's like
https://petstore.swagger.io/?url=https://pulp-python.readthedocs.io/en/latest/_static/api.json#/repositories%3A%20python%20versions/repositories_python_python_versions_repair

If my translation from pulp-speak is correct, is something like `POST /pulp/api/v3/repositories/python/python/aaeab0e1-baf1-482c-b083-fba26f122043`

The other pulp plugin endpoint seem to use the body for the data.  May be more consistent with
pulp to make the src/dest body params.